<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Envio Automático</string>
  </property>
  <property name="windowIcon">
   <iconset resource="file.qrc">
    <normaloff>:/icone/Images/icone.png</normaloff>:/icone/Images/icone.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFFFFF;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Passo 1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="buttonConnectZap">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>368</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
  background-color: #3EB2FD;
  border-radius: 22px;
  border-width: 0;
  color: #FFFFFF;
  font-size: 1rem;
  height: auto;
  line-height: 1.5;
  padding: 6px 20px;
  position: relative;
  text-align: center;
  vertical-align: top;
}

QPushButton:disabled {
  opacity: .3;
}

QPushButton:hover {
  background-image: linear-gradient(#51A9EE, #147BCD);
  border-color: #1482D0;
  text-decoration: none;
}

QPushButton:active {
  background-image: linear-gradient(#3D94D9, #0067B9);
  border-color: #006DBC;
  outline: none;
}

QPushButton:focus {
  outline: none;
}
</string>
            </property>
            <property name="text">
             <string>Conectar Whatsapp</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_17">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Status: </string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelStatusConnectZap">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="text">
                <string>Não Conectado</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Passo 2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_16">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="buttonSelecionaArquivo">
               <property name="font">
                <font>
                 <family>Arial Black</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
  background-color: #3EB2FD;
  border-radius: 22px;
  border-width: 0;
  color: #FFFFFF;
  font-size: 1rem;
  height: auto;
  line-height: 1.5;
  padding: 6px 20px;
  position: relative;
  text-align: center;
  vertical-align: top;
}

QPushButton:disabled {
  opacity: .3;
}

QPushButton:hover {
  background-image: linear-gradient(#51A9EE, #147BCD);
  border-color: #1482D0;
  text-decoration: none;
}

QPushButton:active {
  background-image: linear-gradient(#3D94D9, #0067B9);
  border-color: #006DBC;
  outline: none;
}

QPushButton:focus {
  outline: none;
}
</string>
               </property>
               <property name="text">
                <string>Selecionar Arquivo</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonGeraArquivo">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="font">
                <font>
                 <family>Arial Black</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
  background-color: #3EB2FD;
  border-radius: 22px;
  border-width: 0;
  color: #FFFFFF;
  font-size: 1rem;
  height: auto;
  line-height: 1.5;
  padding: 6px 20px;
  position: relative;
  text-align: center;
  vertical-align: top;
}

QPushButton:disabled {
  opacity: .3;
}

QPushButton:hover {
  background-image: linear-gradient(#51A9EE, #147BCD);
  border-color: #1482D0;
  text-decoration: none;
}

QPushButton:active {
  background-image: linear-gradient(#3D94D9, #0067B9);
  border-color: #006DBC;
  outline: none;
}

QPushButton:focus {
  outline: none;
}
</string>
               </property>
               <property name="text">
                <string>Gerar Arquivo Exemplo (não implementado)</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="default">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_7">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Arquivo Selecionado: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelArquivoSelecionado">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_6">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="maximumSize">
                <size>
                 <width>130</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Total de Contatos na lista: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelTotalContatos">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Tempo Estimado: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelTempoEstimado">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Passo 3:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_15">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Mensagem a enviar:
Já será adiconado ao inicio da mensagem Oi {PrimeiroNome}, tudo bom com você?</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="textoParaEnviar">
            <property name="styleSheet">
             <string notr="true">background-color: #E8F0FE;</string>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhNone</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Passo 4:</string>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_8">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="buttonEnviar">
            <property name="minimumSize">
             <size>
              <width>368</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
  background-color: #3EB2FD;
  border-radius: 22px;
  border-width: 0;
  color: #FFFFFF;
  font-size: 1rem;
  height: auto;
  line-height: 1.5;
  padding: 6px 20px;
  position: relative;
  text-align: center;
  vertical-align: top;
}

QPushButton:disabled {
  opacity: .3;
}

QPushButton:hover {
  background-image: linear-gradient(#51A9EE, #147BCD);
  border-color: #1482D0;
  text-decoration: none;
}

QPushButton:active {
  background-image: linear-gradient(#3D94D9, #0067B9);
  border-color: #006DBC;
  outline: none;
}

QPushButton:focus {
  outline: none;
}
</string>
            </property>
            <property name="text">
             <string>Enviar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_9">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QFrame" name="frame_10">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Enviar Teste para: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditNumeroTeste">
                  <property name="styleSheet">
                   <string notr="true">background-color: #E8F0FE;</string>
                  </property>
                  <property name="inputMethodHints">
                   <set>Qt::ImhPreferNumbers</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonEnviarTeste">
               <property name="font">
                <font>
                 <family>Arial Black</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
  background-color: #3EB2FD;
  border-radius: 22px;
  border-width: 0;
  color: #FFFFFF;
  font-size: 1rem;
  height: auto;
  line-height: 1.5;
  padding: 6px 20px;
  position: relative;
  text-align: center;
  vertical-align: top;
}

QPushButton:disabled {
  opacity: .3;
}

QPushButton:hover {
  background-image: linear-gradient(#51A9EE, #147BCD);
  border-color: #1482D0;
  text-decoration: none;
}

QPushButton:active {
  background-image: linear-gradient(#3D94D9, #0067B9);
  border-color: #006DBC;
  outline: none;
}

QPushButton:focus {
  outline: none;
}
</string>
               </property>
               <property name="text">
                <string>Enviar Teste</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Resultado:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_11">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QFrame" name="frame_12">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QFrame" name="frame_13">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>Total de mensagens enviadas com sucesso: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelEnviadosSucesso">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_14">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="minimumSize">
                   <size>
                    <width>212</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Total de mensagens não enviadas:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelEnviadosErro">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonExportarListaSemNumerosErros">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
  background-color: #3EB2FD;
  border-radius: 22px;
  border-width: 0;
  color: #FFFFFF;
  font-size: 1rem;
  height: auto;
  line-height: 1.5;
  padding: 6px 20px;
  position: relative;
  text-align: center;
  vertical-align: top;
}

QPushButton:disabled {
  opacity: .3;
}

QPushButton:hover {
  background-image: linear-gradient(#51A9EE, #147BCD);
  border-color: #1482D0;
  text-decoration: none;
}

QPushButton:active {
  background-image: linear-gradient(#3D94D9, #0067B9);
  border-color: #006DBC;
  outline: none;
}

QPushButton:focus {
  outline: none;
}
</string>
            </property>
            <property name="text">
             <string>Exporta Lista sem Números com erro(não implementado)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #3EB2FD;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>160000</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image:  0 0 0 0 stretch stretch;
border-image: url(:/iconeFooter/Images/iconeFooter.png);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>SENAI &quot;Almirante Tamandaré&quot;</string>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>versão 0.9.2
Desenvolvido por Eliabel Barreto</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="file.qrc"/>
 </resources>
 <connections/>
</ui>
